.github/workflows/testBuild.yml

```diff
@@ -0,0 +1,178 @@
+name: Build and deploy apps for testing
+
+on:
+  pull_request_target:
+    types: [opened, synchronize]
+
+env:
+  DEVELOPER_DIR: /Applications/Xcode_14.0.1.app/Contents/Developer
+
+jobs:
+  validateActor:
+    runs-on: ubuntu-latest
+    outputs:
+      IS_TEAM_MEMBER: ${{ fromJSON(steps.isUserDeployer.outputs.isTeamMember) }}
+    steps:
+      - id: isUserDeployer
+        uses: tspascoal/get-user-teams-membership@baf2e6adf4c3b897bd65a7e3184305c165aec872
+        with:
+          GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
+          username: ${{ github.actor }}
+          team: mobile-deployers
+
+  android:
+    name: Build and deploy Android
+    needs: validateActor
+    if: ${{ fromJSON(needs.validateActor.outputs.IS_TEAM_MEMBER) }}
```

Comment:
@staszekscp These will ensure the builds will run only on pull_request_target from internal mobile deployers right?

Reply:
That's exactly the reason. For now I would like to make sure that everything works, and next we could establish a list of external contributors for whom the workflow could be triggered.

Reply:
Yeah, we have used this before so this should work just fine.